packages/core-contracts/test/proxy/UniversalProxy.test.js

```diff
@@ -0,0 +1,142 @@
+const { ethers } = hre;
+const assert = require('assert');
+const { assertRevert } = require('@synthetixio/core-js/utils/assertions');
+const { findEvent } = require('@synthetixio/core-js/utils/events');
+
+describe('UniversalProxy', () => {
+  let ForwardingProxy, Instance, Implementation;
```

Comment:
Could we rename `ForwardingProxy` both in this test and in the other test? Not the contract name, but how we call them in the test. I think it will make the tests more readable.

We don't really care what kind of proxy this is, just that it is a proxy.